api/sql/schema/450-rule.falsePropertyFactorCount.sql

```diff
@@ -26,7 +26,7 @@ BEGIN
             JOIN
                 [rule].conditionProperty ct ON ct.name = t.name AND ct.value = t.value
             WHERE
-                p.factor IN ('ss', 'ds', 'cs', 'oc', 'sc', 'dc') AND
+                p.factor IN ('ss', 'ds', 'cs', 'oc', 'sc', 'dc', 'sp', 'dp') AND
```

Comment:
why here? I think it should be in conditionItem?

Reply:
It is fixed with the last commit